observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,62 @@
+* For shorter term data retention, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring-persistent-storage_configuring-the-monitoring-stack[configure persistent storage] for Prometheus and Alertmanager to store metrics and alert data.
+Specify the metrics data retention parameters for Prometheus and Thanos Ruler.
++
+[NOTE]
+====
+By default, in a newly installed {product-title} system, the monitoring `ClusterOperator` resource reports a `PrometheusDataPersistenceNotConfigured` status message to remind you that storage is not configured.
+====
++
+* For longer term data retention, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring_remote_write_storage_configuring-the-monitoring-stack[configure the remote write feature] to enable Prometheus to send ingested metrics to remote systems for storage.
++
+[IMPORTANT]
+====
+Be sure to xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#adding-cluster-id-labels-to-metrics_configuring-the-monitoring-stack[add cluster ID labels to metrics] for use with your remote write storage configuration.
+====
++
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign monitoring cluster roles] to any non-administrator users that need to access certain monitoring features.
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#assigning-tolerations-to-monitoring-components_configuring-the-monitoring-stack[Assign tolerations] to monitoring stack components to enable moving them to tainted nodes.
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#setting-the-body-size-limit-for-metrics-scraping_configuring-the-monitoring-stack[Set the body size limit] for metrics collection to help avoid situations in which Prometheus consumes excessive amounts of memory when scraped targets return a response that contains a large amount of data.
+* xref:../../observability/monitoring/managing-alerts.adoc#managing-core-platform-alerting-rules_managing-alerts[Modify or create alerting rules] for your cluster.
+These rules specify the conditions that trigger alerts, such as high CPU or memory usage, network latency, and so forth.
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#managing-cpu-and-memory-resources-for-monitoring-components[Specify resource limits and requests for monitoring components] to ensure that the containers that run monitoring components have enough CPU and memory resources.
+* xref:../../observability/monitoring/managing-metrics.adoc#managing-metrics[Configure service monitors], which define the set of services and endpoints that Prometheus scrapes for metrics.
+These resources specify the labels that identify the target services and the scraping configuration, including the endpoint paths and ports to scrape.
```

Comment:
I feel like this part adds confusion in this section. We explicitly state this is unsupported for the platform stack (https://docs.openshift.com/container-platform/4.16/observability/monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack) and the linked section talks about UWM.
Is there any reason to not remove this bit?

Reply:
Excellent point. Removed the item from the list